testing/trino-testing/src/main/java/io/trino/testing/AbstractTestQueryFramework.java

```diff
@@ -109,6 +110,15 @@ public void close()
         queryAssertions = null;
     }
 
+    @Test
+    public void ensureTestNamingConvention()
+    {
+        // Enforce a naming convention to make code navigation easier.
+        assertThat(getClass().getName())
+                .doesNotEndWith("ConnectorTest")
```

Comment:
Doesn't work in all cases. What about `BaseMySqlConnectorTest` or something similar?

Reply:
Do you mean to verify all the intermediate classes?
i thought about this and considered there may be some cases where this is not desired, but maybe...

Reply:
Nevermind, the base classes won't ever be instantiated 🤦

Reply:
They won't be, but we _could_ want to enforce naming convention for them as well.

Just that I am _not_ doing this within this PR.